frontend/src/modules/editor/components/KeyboardShortcuts.js

```diff
@@ -0,0 +1,210 @@
+
+
+type State = {|
+    visible: boolean,
+|};
+
+
+/*
+ * Shows a list of keyboard shortcuts.
+ */
+export class EditorSettingsBase extends React.Component<Props, State> {
+    constructor() {
+        super();
+        this.state = {
+            visible: false,
+        };
+    }
+
+    toggleVisibility = () => {
+        this.setState((state) => {
+            return { visible: !state.visible };
+        });
+    }
+
+    // This method is called by the Higher-Order Component `onClickOutside`
+    // when a user clicks outside the keyboard shortcut overlay.
+    handleClickOutside = () => {
+        this.setState({
+            visible: false,
+        });
+    }
+
+    render() {
+        return <div className="keyboard-shortcuts">
+            <Localized
+                id="editor-keyboard-shortcuts-button"
+                attrs={{ title: true }}
+            >
+                <div
+                    className="selector far fa-keyboard"
+                    title="Keyboard Shortcuts"
+                    onClick={ this.toggleVisibility }
+                />
+            </Localized>
+
+            { !this.state.visible ? null :
+            <div
+                className="overlay"
+                onClick={ this.toggleVisibility }
+            >
+                <Localized id="editor-keyboard-shortcuts-overlay-title">
+                    <h2>Keyboard Shortcuts</h2>
+                </Localized>
+                <table>
+                    <tbody>
+                        <tr>
+                            <Localized id="editor-keyboard-shortcuts-save-translation">
+                                <td>Save Translation</td>
+                            </Localized>
+                            <td>
+                                <Localized id="editor-keyboard-shortcuts-save-translation-shortcut">
+                                    <span>Enter</span>
+                                </Localized>
+                            </td>
+                        </tr>
+                        <tr>
+                            <Localized id="editor-keyboard-shortcuts-cancel-translation">
+                                <td>Cancel Translation</td>
+                            </Localized>
+                            <td>
+                                <Localized id="editor-keyboard-shortcuts-cancel-translation-shortcut">
+                                    <span>Esc</span>
+                                </Localized>
+                            </td>
+                        </tr>
+                        <tr>
+                            <Localized id="editor-keyboard-shortcuts-insert-a-new-line">
+                                <td>Insert A New Line</td>
+                            </Localized>
+                            <td>
+                                <Localized
+                                    id="editor-keyboard-shortcuts-insert-a-new-line-shortcut"
+                                    mod1={ <span/> }
```

Comment:
I think the problem here might be that you need to write those tags with a closing tag instead of the autoclosing form, so `<span></span>`.

Reply:
Actually, `<span/>` should work fine.

The problem here is that `Localized` only runs the overlay logic when it wraps an _element_. Here, it wraps a _string_. The solution is to switch the order of the nesting to:

```js
<Localized id="editor-keyboard-shortcuts-insert-a-new-line-shortcut"
    mod1={ <span/> }
    accel={ <span/> }
>
    <td>
        { '<mod1>Shift</mod1> + <accel>Enter</accel>' }
    </td>
</Localized>
```

Sorry about this obscure catch. I'll file an issue about this in `fluent-react`.

Reply:
I filed https://github.com/projectfluent/fluent.js/issues/339 in `fluent-react`.